### PR TITLE
Add Crashlytics 0.0.1 podspec to test private staging

### PR DESCRIPTION
**Pull request by jasonhu-g**

### CODE REVIEW

File: FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec [hunk 1]
```diff
@@ -0,0 +1,82 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseCrashlytics'
+  s.version          = '0.0.1'
+  s.summary          = 'Best and lightest-weight crash reporting for mobile, desktop and tvOS.'
+
+  s.description      = <<-DESC
+Best and lightest-weight crash reporting for mobile, desktop and tvOS TBD-longer.
```
Comment (paulb777) [inline FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec]: Better long description needed.

File: FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                       DESC
+
+  s.homepage         = 'https://developers.google.com/'
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
```
Comment (paulb777) [inline FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec]: Wrong repo

**Review by paulb777 — CHANGES_REQUESTED** [review]
Run pod spec lint before publishing the podspec

File: FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec (hunk 1 above)
Comment (jasonhu-g) [reply FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec]: Done

File: FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec (hunk 2 above)
Comment (jasonhu-g) [reply FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec]: Updated

File: FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec [hunk 3]
```diff
@@ -0,0 +1,82 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseCrashlytics'
+  s.version          = '0.0.1'
+  s.summary          = 'Best and lightest-weight crash reporting for mobile, desktop and tvOS.'
+
+  s.description      = <<-DESC
+Firebase Crashlytics helps you track, prioritize, and fix stability issues that erode app quality, in realtime. Spend less time triaging and troubleshooting crashes and more time building app features that delight users.
```
Comment (paulb777) [inline FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec]: Keep lines less than 100

**Review by paulb777 — APPROVED** [review]
LGTM

File: FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec (hunk 3 above)
Comment (jasonhu-g) [reply FirebaseCrashlytics/0.0.1/FirebaseCrashlytics.podspec]: Done

**Pull request merged**